ci-operator/config/opendatahub-io/maas-billing/opendatahub-io-maas-billing-main.yaml

```
@@ -0,0 +1,44 @@
+build_root:
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: golang-1.24
+images:
+- context_dir: maas-api/
+  dockerfile_path: Dockerfile
+  to: maas-billing
```

Let's use `maas-api`

```suggestion
  to: maas-api
```